### ISSUE TITLE
A11y_api-catalog_.NET API Catalog_Keyboard : "User login Information" drop down is not accessible using keyboard

### ISSUE THREAD

**Issue by SaisreeDandamudi** [terrajobst/apisof.net#7]
[Check out Accessibility Insights] - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS2.1.1; #Keyboard; #C+E-H; #api-catalog; #Edge; #Chrome; #Benchmark; #A11ySev1;

**Environment Details:**
URL: [https://apisof.net/](https://apisof.net/)

**Browser:**
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Repro Steps:**
1. Hit the [URL](https://apisof.net/)
2. Verify whether user is able to access "User login Information" drop down using keyboard

**Actual Result:**
"User login Information" drop down is not accessible using keyboard

**Expected result:**
"User login Information" drop down should be accessible using keyboard

**User Impact:**
Keyboard only users will not get to access "User Login Information" drop down if it is not accessible using keyboard

**Attachment**
[7.login information not accessible using keyboard.pptx](https://github.com/terrajobst/apisof.net/files/3523801/login.information.not.accessible.using.keyboard.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
